lisa/sut_orchestrator/azure/features.py

```diff
@@ -314,6 +315,10 @@ def attach_nics(
         self._log.debug(f"attach the nics into VM {self._node.name} successfully.")
         startstop.start()
 
+    def reload_netvsc(self) -> None:
+        modprobe_tool = self._node.tools[Modprobe]
+        modprobe_tool.reload(["hv_netvsc"])
```

Comment:
Check if it works on sriov. Lili met some problems that there are mlx driver dependencies, so it cannot be reloaded by itself. I'm not sure if they are related.

Reply:
This test passed with SRIOV nic.

2021-11-18 22:24:22.792[27388][DEBUG] lisa.runner[0] Runner finished in 229.503 sec.
2021-11-18 22:24:22.797[27388][DEBUG] lisa.RootRunner running count: 0, id: []
2021-11-18 22:24:22.797[27388][DEBUG] lisa.RootRunner running count: 0, id: []
2021-11-18 22:24:22.797[27388][DEBUG] lisa.RootRunner Idle worker available but no new runner...
2021-11-18 22:24:22.802[27388][INFO] lisa.RootRunner ________________________________________
2021-11-18 22:24:22.802[27388][INFO] lisa.RootRunner NetInterface.validate_network_interface_ifup_ifdown: PASSED
2021-11-18 22:24:22.802[27388][INFO] lisa.RootRunner test result summary
2021-11-18 22:24:22.802[27388][INFO] lisa.RootRunner     TOTAL    : 1
2021-11-18 22:24:22.802[27388][INFO] lisa.RootRunner     QUEUED   : 0
2021-11-18 22:24:22.807[27388][INFO] lisa.RootRunner     ASSIGNED : 0
2021-11-18 22:24:22.812[27388][INFO] lisa.RootRunner     RUNNING  : 0
2021-11-18 22:24:22.817[27388][INFO] lisa.RootRunner     FAILED   : 0
2021-11-18 22:24:22.817[27388][INFO] lisa.RootRunner     PASSED   : 1
2021-11-18 22:24:22.817[27388][INFO] lisa.RootRunner     SKIPPED  : 0
2021-11-18 22:24:22.817[27388][INFO] lisa. completed in 231.823 sec